baseplate/context/cassandra.py

```diff
@@ -3,9 +3,42 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-from cassandra.cluster import _NOT_SET
+from cassandra.cluster import Cluster, _NOT_SET
 
 from . import ContextFactory
+from .. import config
+
+
+def cluster_from_config(app_config, prefix="cassandra.", **kwargs):
```

Comment:
If there is not already test coverage on `cluster_from_config` and `pool_from_config`, a couple small unit tests would be good.